webauthn/authentication/verify_authentication_response.py

```diff
@@ -67,6 +68,8 @@ def verify_authentication_response(
     Raises:
         `helpers.exceptions.InvalidAuthenticationResponse` if the response cannot be verified
     """
+    if isinstance(credential, (str, bytes, bytearray)):
```

Comment:
This will need a similar treatment as below 😊

```suggestion
   if isinstance(credential, str):
```

Reply:
and that one too.